packages/components/pie-radio-group/src/index.ts

```diff
@@ -11,10 +25,133 @@ const componentSelector = 'pie-radio-group';
 
 /**
  * @tagname pie-radio-group
+ * @slot - Default slot
+ * @slot label - The label slot
+ * @event {CustomEvent} change - when one of the radios state is changed.
  */
-export class PieRadioGroup extends LitElement implements RadioGroupProps {
+export class PieRadioGroup extends FormControlMixin(RtlMixin(LitElement)) implements RadioGroupProps {
+    @state()
+        hasLabel = false;
+
+    @property({ type: String })
+    public name?: RadioGroupProps['name'];
+
+    @property({ type: String })
+    public value = defaultProps.value;
+
+    @property({ type: Boolean })
+    public isInline = defaultProps.isInline;
+
+    @property({ type: Boolean, reflect: true })
+    public disabled = defaultProps.disabled;
+
+    @queryAssignedElements({ selector: 'pie-radio' })
+        _slottedChildren: Array<HTMLInputElement> | undefined;
```

Comment:
Please could you confirm whether this returns an empty array or undefined (or something else) if there are no pie-radio children?

Reply:
My thinking is that if it always returns an array (either empty or containing elements) then we can add `!` to avoid needing to do `this._slottedChildren?`, etc

Reply:
it's an empty array 🤝 

<img width="285" alt="Screenshot 2024-10-25 at 10 46 34" src="https://github.com/user-attachments/assets/68b34761-7dda-48f0-ae5e-eab7cc838385">

Reply:
updated the checkbox group too 👍